internal/ui/settings_ui.go

```diff
@@ -361,3 +371,13 @@ func modifierToString(mods deskDriver.Modifier) string {
 	}
 	return strings.Join(s, "+")
 }
+
+func getPicturesDir() (fyne.ListableURI, error) {
+	home, err := os.UserHomeDir()
```

Comment:
To do this correctly will be more work I'm afraid.
The Pictures directory is part of the same XDG spec that is used to find the correct locations for the favourites folders...

`XDG_PICTURES_DIR="$HOME/Pictures"`

I don't think that code was exported from the Fyne `dialog` package in any way, so you may need to re-do the `xdg-user-dirs` lookup yourself here.

Reply:
Good point. Fixed now :)